packages/devtools-reps/src/object-inspector/utils/node.js

```diff
@@ -877,6 +877,24 @@ function getClosestNonBucketNode(item: Node): Node | null {
   return getClosestNonBucketNode(parent);
 }
 
+function getNonPrototypeParentGripValue(item: Node | null): Node | null {
+  const parentNode = getParent(item);
+  if (!parentNode) {
+    return null;
+  }
+
+  const parentGripNode = getClosestGripNode(parentNode);
+  if (!parentGripNode) {
+    return null;
+  }
+
+  if (getType(parentGripNode) === NODE_TYPES.PROTOTYPE) {
+    return getNonPrototypeParentGripValue(parentGripNode);
+  }
+
+  return getValue(parentGripNode);
+}
+
 function getParentGripValue(
```

Comment:
I don't think getParentGripValue is used anywhere anymore?

Reply:
good catch. Let me remove it :)